<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>My Notes</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QCalendarWidget" name="MyCalendar">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>391</width>
      <height>461</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="firstDayOfWeek">
     <enum>Qt::Sunday</enum>
    </property>
    <property name="gridVisible">
     <bool>true</bool>
    </property>
    <property name="verticalHeaderFormat">
     <enum>QCalendarWidget::ISOWeekNumbers</enum>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="NewNote">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>20</y>
      <width>351</width>
      <height>461</height>
     </rect>
    </property>
    <property name="plainText">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="AddNote">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>500</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Add note</string>
    </property>
   </widget>
   <widget class="QPushButton" name="DeleteNote">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>500</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete note</string>
    </property>
   </widget>
   <widget class="QLabel" name="Text">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>530</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuNotes">
    <property name="title">
     <string>Notes</string>
    </property>
   </widget>
   <addaction name="menuNotes"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
